### ISSUE TITLE
Update to incorporate sklearn api change

### ISSUE THREAD

**Issue by acwooding** [ddangelov/Top2Vec#318]
In sklearn 1.2 `.get_feature_names` is now deprecated in favour of `.get_feature_names_out()` (see https://github.com/scikit-learn/scikit-learn/commit/2708aac0348402850f8e5b3176e84b39c6f5917d). 

This leads to the following error:
```
Top2Vec.py:581, in Top2Vec.__init__(self, documents, min_count, ngram_vocab, ngram_vocab_args, embedding_model, embedding_model_path, embedding_batch_size, split_documents, document_chunker, chunk_length, max_num_chunks, chunk_overlap_ratio, chunk_len_coverage_ratio, sentencizer, speed, use_corpus_file, document_ids, keep_documents, workers, tokenizer, use_embedding_model_tokenizer, umap_args, hdbscan_args, verbose)
    579 vectorizer = CountVectorizer(tokenizer=return_doc, preprocessor=return_doc)
    580 doc_word_counts = vectorizer.fit_transform(tokenized_corpus)
--> 581 words = vectorizer.get_feature_names()
    582 word_counts = np.array(np.sum(doc_word_counts, axis=0).tolist()[0])
    583 vocab_inds = np.where(word_counts > min_count)[0]

AttributeError: 'CountVectorizer' object has no attribute 'get_feature_names'
```

Either require sklearn < 1.2 or update the api (I'd recommend updating the api).

**Comment by ddangelov** [comment]
Update coming later today! Thank you for the recommendation! :)

**Comment by ddangelov** [comment]
Fixed in version 1.0.28!
